Makefile

```diff
@@ -12,6 +12,7 @@ clean: clean_openwrt clean_feeds clean_binaries clean_keys
 # Building OpenWRT
 $(DIR__CI)/patched:
 	git submodule init openwrt;git submodule update --remote; \
+	cp $(DIR__CI)/feeds.conf.default $(DIR__OPENWRT)/feeds.conf.default; \
```

Comment:
I have changed the filename to feeds.conf.creator.ci and when its copied it gets the name feeds.conf